src/core/annotation.js

```diff
@@ -1013,6 +1016,32 @@ class PolygonAnnotation extends PolylineAnnotation {
   }
 }
 
+class InkAnnotation extends Annotation {
+  constructor(parameters) {
+    super(parameters);
+
+    this.data.annotationType = AnnotationType.INK;
+
+    let dict = parameters.dict;
+    let rawInkLists = dict.getArray('InkList');
```

Comment:
Using "raw" in this name is somewhat misleading, since `getArray` will dereference the top level elements when fetching the data (which is very different to `getRaw`); please see the `Dict` class for additional details: https://github.com/mozilla/pdf.js/blob/master/src/core/primitives.js#L114-L134.

Could another name be used instead?
As always, naming things is one of the hard parts of computer science, but how about e.g. `originalInkLists` or `basicInkLists` or something instead!?